server.py

```diff
@@ -169,7 +170,14 @@ def do_get_responses():
         responses.append(document)
 
     results['results'] = responses
-    return jsonify(results)
+    return Response(json.dumps(results, default=json_serial))
+
+
+def json_serial(obj):
```

Comment:
I think this method should be before the route methods rather than in between them